pgatk/commands/clinvar_to_proteindb.py

```
@@ -0,0 +1,29 @@
+from pgatk.clinvar.clinvar_service import ClinVarService
+
+
+@click.command("clinvar-to-proteindb", short_help="Generate protein database from ClinVar VCF + RefSeq GTF")
+@click.option("-c", "--config_file", help="Configuration YAML file (optional, uses bundled defaults)")
+@click.option("-v", "--vcf", required=True, help="ClinVar VCF file path")
+@click.option("-g", "--gtf", required=True, help="NCBI RefSeq GTF file path")
+@click.option("-f", "--fasta", required=True, help="RefSeq transcript nucleotide FASTA file path")
+@click.option("-a", "--assembly-report", required=True, help="NCBI assembly report file path")
+@click.option("-o", "--output", required=True, help="Output protein FASTA file path")
+@click.pass_context
+def clinvar_to_proteindb(ctx, config_file, vcf, gtf, fasta, assembly_report, output):
```

Ruff (0.15.2)

[warning] Unused function argument: `ctx`

(ARG001)